src/main/java/com/greenfox/tribesoflagopus/backend/model/entity/User.java

```diff
@@ -13,27 +8,34 @@
 import javax.persistence.OneToOne;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Entity
-@Builder
 @Getter
 @Setter
-public class Player {
+@NoArgsConstructor
+public class User {
 
   @Id
   @GeneratedValue(strategy = GenerationType.AUTO)
   private long id;
+
   private String username;
 
-  @OneToOne(mappedBy = "player", cascade = CascadeType.ALL, orphanRemoval = true)
-  @JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
-  @JsonIdentityReference(alwaysAsId = true)
-  @JsonProperty(value = "kingdomId")
+  @OneToOne(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
   private Kingdom kingdom;
```

Comment:
As we discussed this today, you should change this `Kingdom kingdom` field to `long kingdomId` for your next pull request.